shared/js/ui/templates/whitelist-items.es6.js

```diff
@@ -0,0 +1,13 @@
+const bel = require('bel')
```

Comment:
It's easier to rerender this list selectively if I take it out in another template.